### ISSUE TITLE
Type calculus

### ISSUE THREAD

**Issue by alexraj123** [skarger/compiler#1]
Hi, I was browsing through your code for Compiler, it looks great. One thing I noticed though is the missing code for type calculus once the symbol table is created. Have you not done that?

Regards
Alex

**Comment by skarger** [comment]
@alexraj123 Correct, I didn't get to the type handling. If you're taking the course I'd advise asking about it during office hours. I found that the most helpful for getting through the problem sets.
